ibm/resource_ibm_private_dns_glb_monitor.go

```diff
@@ -0,0 +1,529 @@
+			},
+
+			pdnsGlbMonitorPort: {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Description: "Port number to connect to for the health check",
+			},
+
+			pdnsGlbMonitorInterval: {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Default:     60,
+				Description: "The interval between each health check",
+			},
+
+			pdnsGlbMonitorRetries: {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Default:     1,
+				Description: "The number of retries to attempt in case of a timeout before marking the origin as unhealthy",
+			},
+
+			pdnsGlbMonitorTimeout: {
+				Type:        schema.TypeInt,
+				Optional:    true,
+				Default:     5,
+				Description: "The timeout (in seconds) before marking the health check as failed",
+			},
+
+			pdnsGlbMonitorMethod: {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validateAllowedStringValue([]string{"GET", "HEAD"}),
+				Description:  "The method to use for the health check",
+			},
+
+			pdnsGlbMonitorPath: {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "The endpoint path to health check against",
+			},
+
+			pdnsGlbMonitorHeaders: {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						pdnsGlbMonitorHeadersName: {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+
+						pdnsGlbMonitorHeadersValue: {
+							Type:     schema.TypeList,
+							Required: true,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+							},
+						},
+					},
+				},
+			},
+
+			pdnsGlbMonitorAllowInsecure: {
+				Type:        schema.TypeBool,
+				Optional:    true,
+				Description: "Do not validate the certificate when monitor use HTTPS. This parameter is currently only valid for HTTPS monitors.",
+			},
+
+			pdnsGlbMonitorExpectedCodes: {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validateAllowedStringValue([]string{"200", "201", "202", "203", "204", "205", "206", "207", "208", "226", "2xx"}),
+				Description:  "The expected HTTP response code or code range of the health check. This parameter is only valid for HTTP and HTTPS",
+			},
+
+			pdnsGlbMonitorExpectedBody: {
+				Type:        schema.TypeString,
+				Optional:    true,
+				Computed:    true,
+				Description: "A case-insensitive sub-string to look for in the response body",
+			},
+
+			pdnsGlbMonitorCreatedOn: {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "GLB Monitor creation date",
+			},
+
+			pdnsGlbMonitorModifiedOn: {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "GLB Monitor Modification date",
+			},
+		},
+	}
+}
+
+func resourceIBMPrivateDNSGLBMonitorCreate(d *schema.ResourceData, meta interface{}) error {
+	sess, err := meta.(ClientSession).PrivateDnsClientSession()
```

Comment:
I do not see ibm/config.go file in this diff.  Can you please check and let me know where is it?

Reply:
Config files changes are already merged for PDNS changes